website/docs/r/cloudsearch_domain.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "CloudSearch"
+layout: "aws"
+page_title: "AWS: aws_cloudsearch_domain"
+description: |-
+  Provides an CloudSearch domain resource. 
+---
+
+# Resource: aws_cloudsearch_domain
+
+Provides an CloudSearch domain resource.
+
+## Example Usage
+
+```terraform
+resource "aws_cloudsearch_domain" "my_domain" {
+  domain_name   = "test-domain"
```

Comment:
I think this should be `name` instead of `domain_name`

Reply:
Thanks, good catch!